tests/Validator/PhoneNumberValidatorTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\EoneoPay\Externals\Validator;
+
+use EoneoPay\Externals\Validator\PhoneNumberValidator;
+use Tests\EoneoPay\Externals\TestCase;
+
+/**
+ * @covers \EoneoPay\Externals\Validator\PhoneNumberValidator
+ */
+class PhoneNumberValidatorTest extends TestCase
+{
+    /**
+     * Returns test data for format function
+     *
+     * @return mixed[]
+     */
+    public function getFormatData(): iterable
+    {
+        yield ['0400000000', '+61400000000'];
+        yield ['+61400000000', '+61400000000'];
+        yield ['elephant', null];
+    }
+
+    /**
+     * Returns test data for the validate function.
+     *
+     * @return mixed[]
+     */
+    public function getValidityData(): iterable
+    {
+        yield ['0400000000', true];
```

Comment:
Needs to also add;
toll free numbers, landlines, extensions. Phone numbers with between 8, 9, and 10 digits.

Reply:
Numbers with 8 digits are not valid - an area code is never implicit in a system like this.

libphonenumber does not emit extension details when using E164 formatting, so I think we should probably just not support it either.

Reply:
https://github.com/giggsey/libphonenumber-for-php/issues/152